### PR TITLE
Restyled the page and added the upload icon.

### PR DESCRIPTION
**Pull request by stevieing**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

**Comment by stevieing** [comment]
Screenshot 2023-02-24 at 11 11 42

File: src/components/qcResultsUpload/QcResultsUploadForm.vue [hunk 1]
```diff
@@ -1,54 +1,64 @@
...
+      ></traction-file>
+      <div class="space-x-4 pb-4 flex flex-row justify-end">
+        <traction-button
+          id="upload-button"
+          type="submit"
+          theme="create"
+          size="lg"
+          :disabled="disableUpload"
+        >
+          <!-- Weird bug - spinner won't show unless button text is two words/ big enough? -->
```
Comment (BenTopping) [inline src/components/qcResultsUpload/QcResultsUploadForm.vue:34]: I think this bug has been fixed. Worth testing and removing comment if it is.

File: src/components/qcResultsUpload/QcResultsUploadForm.vue [hunk 2]
```diff
@@ -1,54 +1,64 @@
...
+          Upload File
+          <UploadIcon class="pl-1" />
+          <traction-spinner v-show="busy"></traction-spinner>
```
Comment (BenTopping) [inline src/components/qcResultsUpload/QcResultsUploadForm.vue:37]: Its not really an issue but I noticed the spinner does cover the UploadIcon when active. Maybe have the spinner replace the UploadIcon conditionally or might be nice to replace the text and icon conditionally.

**Review by BenTopping — CHANGES_REQUESTED** [review]
Page looks really good. ESLint has some minor warnings (wondering if we should fail the CI on warnings) and a small detail to check about the spinner.

**Review by BenTopping — APPROVED** [review]

**Pull request merged**